test/e2e/common/configmap.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package common
 
 import (
+	"encoding/json"
 	"fmt"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
need to change import order like
```
diff -u ./test/e2e/common/configmap.go.orig ./test/e2e/common/configmap.go
--- ./test/e2e/common/configmap.go.orig	2020-01-14 22:30:26.136435007 +0000
+++ ./test/e2e/common/configmap.go	2020-01-14 22:30:26.136435007 +0000
@@ -22,10 +22,10 @@
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
-	"k8s.io/apimachinery/pkg/types"
 
 	"github.com/onsi/ginkgo"
 )
```
for passing pull-kubernetes-verify job.

Reply:
Updated in https://github.com/kubernetes/kubernetes/pull/87212/commits/f20945aa416bc8dae263d0e31a00f54d677946f3